src/components/RepositoriesCards.tsx

```
@@ -1,16 +1,33 @@
 import { RepositoriesCardsProps } from "../types/RepositoriesCardsProps";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { faExternalLinkAlt, faStar } from "@fortawesome/free-solid-svg-icons";
+import {
+  faExternalLinkAlt,
+  faStar,
+  faClock,
+} from "@fortawesome/free-solid-svg-icons";
 import "./RepositoriesCards.css";
+const formatDate = (isoDate: string): string => {
+  const date = new Date(isoDate);
+  if (isNaN(date.valueOf())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)